src/styles/dgrid.css

```diff
@@ -1 +1,2 @@
+@import 'headerCell.css';
```

Comment:
I'm not sure how to exclude this from the poscss task. Right now I'm using the pattern from dojo/widgets that just overwrites the file with its original version.